.pre-commit-config.yaml

```diff
@@ -494,19 +494,25 @@ repos:
         language: system
         entry: "./scripts/ci/pre_commit/pre_commit_mypy.sh"
         files: \.py$
-        exclude: ^dev|^provider_packages|^chart
+        exclude: ^dev|^provider_packages|^chart|^docs
       - id: mypy
         name: Run mypy for helm chart tests
         language: system
         entry: "./scripts/ci/pre_commit/pre_commit_mypy.sh"
         files: ^chart/.*\.py$
         require_serial: true
+      - id: mypy
```

Comment:
mypy does not support running checks a module (`mypy -m airflow`) and a file (`mypy dcos/ext /`) at the same run.